### PR TITLE
Implementing the delegation modal UI

### PR DESCRIPTION
**Pull request by stanfilip0v**
Creating the delegation modal UI

### CODE REVIEW

File: src/containers/Staking/components/Validators/components/DelegationModal/Failure.tsx [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+        flexDirection="column"
+        alignItems="center"
+        textAlign="center"
+        gap={1}
+      >
+        <Typography variant="h4" fontWeight={900} letterSpacing={2}>
+          Delegation failed!
+        </Typography>
+        <Typography variant="subtitle1" color="text.secondary">
+          Seems like something went wrong with executing the transaction . Try
```
Comment (SpaghettiOverload) [inline src/containers/Staking/components/Validators/components/DelegationModal/Failure.tsx]: Typo on line 36

File: src/containers/Staking/components/Validators/components/DelegationModal/Success.tsx [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+    validator && (
+      <ModalContainer sx={{ padding: '4rem' }}>
+        <img src={SuccessIcon} alt="success-icon" />
+        <CancelRoundedIcon onClick={handleClose} />
+        <Box display="flex" flexDirection="column" alignItems="center" gap={1}>
+          <Typography variant="h4" fontWeight={900} letterSpacing={2}>
+            Success!
+          </Typography>
+          <Typography variant="subtitle1" color="text.secondary">
+            Transaction was successfully executed
```
Comment (SpaghettiOverload) [inline src/containers/Staking/components/Validators/components/DelegationModal/Success.tsx]: Maybe an exclamation mark on the end of line 35.

File: src/containers/Staking/components/Validators/components/DelegationModal/Failure.tsx (hunk 1 above)
Comment (stanfilip0v) [reply src/containers/Staking/components/Validators/components/DelegationModal/Failure.tsx]: fixed

File: src/containers/Staking/components/Validators/components/DelegationModal/Success.tsx (hunk 2 above)
Comment (stanfilip0v) [reply src/containers/Staking/components/Validators/components/DelegationModal/Success.tsx]: fixed

**Review by SpaghettiOverload — APPROVED** [review]

**Review by stefanpetkov90 — APPROVED** [review]

**Pull request merged**